src/browser.ts

```diff
@@ -187,6 +188,8 @@ export class GrpcClient {
 
       const url = `${grpcFallbackProtocol}://${servicePath}:${servicePort}/$rpc/${protoServiceName}/${rpcName}`;
 
+      const nodeFetch = require('node-fetch');
```

Comment:
since it's TypeScript, can we `import * as nodeFetch from 'node-fetch'` instead? (on top of the file)

Also, please make sure `node-fetch` is in `package.json`.

Reply:
Example of the similar code in another package we use: https://github.com/googleapis/gaxios/blob/19b4b3c7dffb0f01bd475932fab5cc5642a19c23/src/gaxios.ts#L16-L33